tests/unittests/distros/test_netconfig.py

```diff
@@ -459,177 +489,206 @@ def _apply_and_verify_eni(
         self,
         apply_fn,
         config,
+        tmp_path,
         expected_cfgs=None,
         bringup=False,
         previous_files=(),
     ):
         if not expected_cfgs:
             raise ValueError("expected_cfg must not be None")
 
-        tmpd = None
+        tmpd = tmp_path
```

Comment:
Did we just assign the tmpd local to reduce character count to avoid splitting/wrapping lines below?